packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -41,6 +41,7 @@ class DriveCommand extends RunCommandBase {
   DriveCommand() {
     argParser.addFlag(
       'keep-app-running',
+      defaultsTo: null,
```

Comment:
this is the line required to fix the build; the rest is just the debugging logic i added to track down the problem.